main.go

```diff
@@ -425,9 +448,28 @@ func (ps *PubSubStreamer) Close() {
 	ps.topic.Stop()
 }
 
+type StackdriverStreamer struct {
+	logger *stackdriver.Logger
+}
+
+func NewStackdriverStreamer(client *stackdriver.Client) *StackdriverStreamer {
+	l := client.Logger(globalConfig.gcpStackdriverName)
+	return &StackdriverStreamer{logger: l}
+}
+
+func (s *StackdriverStreamer) Send(record map[string]interface{}) {
+	s.logger.Log(stackdriver.Entry{Payload: record})
```

Comment:
I noticed in the docs it says if logging JSON wrap it in `json.RawMessage`, not sure if that is required here or not.

https://godoc.org/cloud.google.com/go/logging#hdr-Payloads

Reply:
> If you have a []byte of JSON, wrap it in json.RawMessage

Oh I missed that. That should allow for this func to use the byte slice like the other `Send()` funcs.